app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+
+  def create
+    @user = User.find_by(id: params[:user_id])
+    @post = Post.find_by(id: params[:post_id])
+    @like = Like.create(like_params)
+    @like.save
+    redirect_to user_path(@user)
```

Comment:
since we will have the like button from two different places maybe it is worth redirecting back to the page they came from.

Reply:
discussed

Reply:
bare this in mind for when you add the like button on the profile page

Reply:
Sure!